gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnAppLauncher.java

```diff
@@ -791,6 +798,20 @@ private void addAppRemoteFiles(String hdfsFileList, Map<String, LocalResource> r
     }
   }
 
+  private void addAppRemoteZips(String hdfsFileList, Map<String, LocalResource> resourceMap)
```

Comment:
Can this method be moved to YarnHelixUtils to avoid code duplication in YarnService?